sycl/source/detail/scheduler/commands.hpp

```diff
@@ -509,8 +509,6 @@ class ExecCGCommand : public Command {
   // memory in DispatchNativeKernel.
   // TODO remove when native kernel support is terminated.
   void releaseCG() {
-    assert(MCommandGroup->getType() == CG::RUN_ON_HOST_INTEL &&
-           "Only 'native kernel' is allowed to release command group");
     MCommandGroup.release();
```

Comment:
This doesn't seem to be related to the intrinsic fix. Please, create separate PR for this change.

Reply:
Removed this change